lib/model.js

```diff
@@ -484,6 +484,7 @@ var validateIncludedElements = function(options, tableNames) {
         include.subQuery = false;
       } else {
         include.subQueryFilter = false;
+        include.subQuery = include.hasParentRequired && (include.hasDuplicating || include.hasRequired);
```

Comment:
This would overwrite the line above that supports an already set `include.subQuery`.
Simply prefix the hasParentRequired check with `include.subQuery || ` and it should be ok.